LIMS_IMAGE/web/templates/orders/shopping.html

```diff
@@ -2,30 +2,85 @@
 <!-- Not having this here makes the interpreter mad -->
 {% block title %}Client Home{% endblock %}
 <!-- Not having this here makes the interpreter mad -->
+<body>  
+    
+    {% block content %}
+
+    <h1>Shopping Cart (Order Submission)</h1> 
+        <br>
+      <p><b> Hello, {{user}} {{account_number}}<br>
+            <br>
+            &emsp; Please complete the order form by selecting your sample type, test type and the quantity for each order. <br>
+            &emsp; If you are not sure which test or package to order please see Appendix B for our recommendation. 
+      </b> </p>   
+        {% if user %} <!--this should be account_number-->
+            <form>
+            <h3> &emsp; Sample submission form</h3>
+            
+                <div class = "container mt-3"> 
+                    <form action = "/action_page.php">
+                        <label for="sample-type" class="form-label">Choose your sample from the list:</label>
+                        <input class="form-control" list="sample-types" name="sample-type" id="sample-type">
+                        <datalist id="sample-types">
+                            <option value="Dairy">
+                            <option value="Cosmetics">
+                            <option value="Environment">
+                        </datalist>   
+                    </form>
+                </div>
 
-<body>
+                <div class = "container mt-3">
+                    <form action = "/action_page.php">
+                        <label for="sample-num" class="form-label"> How many samples of your selected type above you can submit for the testing? </label>
+                        <input class="sample-num" list="sample-nums" name="sample-num" id="sample-num">
+                        <datalist id="sample-nums">
+                            <option>1</option> 
```

Comment:
if you are asking for a number, that can be any value, this should be a text box entry, not a drop down.
Is there a reason the only options are 1,2,3,4+?